js/exporter/jspdf/v3/pdf_utils_v3.js

```diff
@@ -31,9 +31,13 @@ function calculateRowHeight(doc, cells, columnWidths) {
 
     let rowHeight = 0;
     for(let cellIndex = 0; cellIndex < cells.length; cellIndex++) {
-        const cellText = cells[cellIndex].text;
-        const font = cells[cellIndex].font;
-        const wordWrapEnabled = cells[cellIndex].wordWrapEnabled;
+        if(isDefined(cells[cellIndex].rowSpan)) {
```

Comment:
It's not clear whether cells[cellIndex] is 'mergedCellsMaster' or 'mergedCellsSlave' ?
Does it make sense in this condition ?

Reply:
> Does it make sense in this condition ?
Nope, I skip all cells with rowsSpan, including mergedCellsMaster and mergedCellsSlave.
This code computes height only for simple cells without any rowSpan.

Height for the cells with rowSpan will be computer later.